config/config_test.go

```diff
@@ -18,7 +18,7 @@ func TestParseTerragruntConfigDynamoLockMinimalConfig(t *testing.T) {
 	t.Parallel()
 
 	config :=
-`
+		`
```

Comment:
Hm, these changes are annoying and look weird. Is there a cleaner way to format multi-line strings in Go? Perhaps the opening backtick should go on the same line as the `:=`.